### ISSUE TITLE
DNF support

### ISSUE THREAD

**Issue by vjanelle** [choria-plugins/package-agent#14]
CentOS 8 introduces yet-another-package manager, with new APIs and locations for python.

**Comment by vjanelle** [comment]
The only thing I could really think about is the yum_ commands:

```
[root@a5a135be4025 /]# which yum
/usr/bin/yum
[root@a5a135be4025 /]# yum clean all
Failed to set locale, defaulting to C
20 files removed
[root@a5a135be4025 /]# cat /etc/redhat-release
CentOS Linux release 8.0.1905 (Core)
```

Adding search support would require a new bridge, but it's probably a lot of copy and paste between dnf and yum.

**Comment by ripienaar** [comment]
yeah we can keep this open for when we one day get time/inclination but also happy to close if you dont see the need, i dont think anyone use these providers really

**Comment by vjanelle** [comment]
@ripienaar useful if you don't want to have puppet around, I suppose.

**Comment by ripienaar** [comment]
yeah when we did this we also did same for service as we had some interest from chef people to support them .... but it just never panned out :( That said recently quite a few people been asking for non Puppet choria, so been tinkering on a k8s based solution for the shared infra
